docs/manage/threat-intelligence-indicators.md

```diff
@@ -0,0 +1,60 @@
+
+Threat intelligence indicators can help a security analysts leverage a large body of information to surface potential threats. For example, say that a threat intelligence database has an indicator that correlates a certain IP address with known malicious activity. Because of this correlation, analysts can assume log messages with that IP address are more likely to be part of a real cyber attack.
+
+You can add threat intelligence indicators from a number of sources, including CrowdStrike, TAXII, ThreatQ, iDefense, and many others. And threat intelligence indicators imported to Sumo Logic not only integrate with your existing core Sumo Logic deployment, but also Cloud SIEM and Cloud SOAR. (For information about how to add additional threat intelligence indicators for Cloud SIEM, see [Create a Custom Threat Intelligence Source](/docs/cse/administration/create-custom-threat-intel-source).)
+
+## Threat Intelligence tab
+
+Use the **Threat Intelligence** tab to add and manage threat intelligence indicators. To access the tab, go to **Manage Data > Logs > Threat Intelligence**.
+
+<img src={useBaseUrl('img/manage/threat-intelligence-tab.png')} alt="Threat Intelligence tab" style={{border: '1px solid black'}} width="800" />
+
+1. **Add Indicators**. Click to upload files that [add threat intelligence indicators](#add-threat-intelligence-indicators).
+1. **Actions**. Select to perform additional actions:
+    * **Edit Retention Period**. Enter the length of time in days to retain expired threat intelligence indicator files. The maximum number of days is 180.
+1. **Source Name**. The source of the threat intelligence indicator file.
+1. **Storage Consumed**. The amount of storage consumed by the threat intelligence indicator file.
+1. **Indicators**. The number of threat intelligence indicators included in the file. 
+
+## Add threat intelligence indicators
+
+To add threat intelligence indicators, you must upload files containing the indicators in a format that can be consumed by Sumo Logic.
+
+1. In Sumo Logic, go to **Manage Data > Logs > Threat Intelligence**.
+1. Click **Add Indicators**. The dialog displays. <br/><img src={useBaseUrl('img/manage/threat-intelligence-add-indicators.png')} alt="Add threat intelligence indicators" style={{border: '1px solid black'}} width="500" />
+1. Select the format of the file to be uploaded:
+    * **Sumo normalized JSON**. A JSON file [normalized](/docs/cse/schema/parsing-language-reference-guide#normalizing) for use in Sumo Logic. Your file must include the following attributes:
```

Comment:
This is more accurate info: 
https://api.sumologic.com/docs/#operation/uploadNormalizedIndicators

Stix:
https://api.sumologic.com/docs/#operation/uploadStixIndicators
We have a meeting on how to handle stix indicator uploads. From UI perspective only the fields in indicators object which are marked required are required with file upload.

Reply:
@knamburi-sumo, thank you very much for that information! I added links to the APIs with the [Add API links](https://github.com/SumoLogic/sumologic-documentation/pull/2936/commits/010d57b05009ab378f517e67221a77f4cffa016b) commit.